package.json

```diff
@@ -21,11 +23,16 @@
   },
   "homepage": "https://github.com/Aksana-Tsishchanka/npm_task1#readme",
   "devDependencies": {
-    "eslint": "4.6.1"
+    "eslint": "4.6.1",
```

Comment:
Don't know why but it does not work for me. I get `Error: Cannot find module 'escope'`.